rules/xcodeproj.bzl

```diff
@@ -540,6 +541,8 @@ def _populate_xcodeproj_targets_and_schemes(ctx, targets, src_dot_dots, all_tran
             ["-D%s" % d for d in target_info.cc_defines.to_list()],
         )
 
+        target_settings["BAZEL_LLDB_INIT_FILE"] = lldbinit_file
```

Comment:
so this file is where to put content of `lldb-init-source-map` in, thx @thiagohmcruz !

Reply:
Yep! Setting it to an env var here so the shell scripts can read it, same path goes to Xcode's generate scheme action.